MediaContainer Patch

LEPII opened the pull request:
- Added "searchString" dependency;  
- Patched MediaContainer to conditionally render 'found results' or no '……results' depending on the URL; 
- Most of the code seems to be moved but not changed;
- URL will update after merging a different PR (HomePage);

LEPII committed:
patched MediaContainer to conditonally render 'found results' or no 'results' depending on the URL

avecc003 commented:
Looks good! 

Feedback:
- Found x results for {query} is missing the word results.
- Change the conditional "Found" from a ```<p>``` tag to a ```<h1>``` tag to match the title.

LEPII committed:
Added conditional results word and changed p tag to h1.

LEPII commented:
> Looks good!
> 
> Feedback:
> 
> * Found x results for {query} is missing the word results.
> * Change the conditional "Found" from a `<p>` tag to a `<h1>` tag to match the title.

Added conditional results word and changed p tag to h1.

avecc003 commented:
Looks good! Ready to merge.

avecc003 committed:
Merging develop before merging into develop
